universite-du-quebec-a-montreal.csl

```diff
@@ -293,31 +305,30 @@
       </else-if>
       <else-if type="article-journal" match="any">
         <choose>
-          <if match="all" variable="title">
-            <choose>
-              <if match="any" variable="container-title">
-                <group delimiter=" ">
-                  <text variable="title"/>
-                  <text variable="genre" prefix="[" suffix="]"/>
-                </group>
-              </if>
-              <else>
-                <group delimiter=". ">
-                  <text variable="title" font-style="italic"/>
-                  <text variable="genre"/>
-                </group>
-              </else>
-            </choose>
+          <if match="any" variable="container-title">
+            <group delimiter=" ">
+              <text variable="title"/>
+              <text variable="genre" prefix="[" suffix="]"/>
+            </group>
           </if>
+          <else>
+            <group delimiter=". ">
+              <text variable="title" font-style="italic"/>
+              <text variable="genre"/>
+            </group>
+          </else>
         </choose>
       </else-if>
       <else-if type="review-book" match="any">
         <group delimiter=", " prefix="[" suffix="]">
           <text variable="title"/>
           <text variable="reviewed-title" font-style="italic" prefix="Compte rendu du livre "/>
-          <names variable="reviewed-author" prefix="par ">
-            <name and="text" delimiter-precedes-last="never" et-al-min="8" et-al-use-first="6" et-al-subsequent-min="8" et-al-subsequent-use-first="6" et-al-use-last="true" initialize-with="."/>
-          </names>
+          <group delimiter=" ">
+            <text term="by"/>
+            <names variable="reviewed-author">
+              <name and="text" delimiter-precedes-last="never" et-al-min="8" et-al-use-first="6" et-al-subsequent-min="8" et-al-subsequent-use-first="6" et-al-use-last="true" initialize-with="."/>
```

Comment:
no need for the et al's here in `name` -- they'll get inherited from the bibliography element.

Reply:
OK, I will remove it.